config/locales/errors.en-GB.yml

```diff
@@ -41,3 +41,5 @@ en-GB:
       address:
         blank: "Address must be completed"
         too_long: "Address is too long"
+      privacy_notice:
+        blank: "Privacy notice must be checked"
```

Comment:
I think we need to check this wording - probably needs to be something more like "You must agree to our privacy policy"

Reply:
This isn't used now, it comes up with the :accepted error, so just going to delete